torch/nn/modules/loss.py

```diff
@@ -1254,6 +1257,108 @@ def forward(self, anchor: Tensor, positive: Tensor, negative: Tensor) -> Tensor:
                                      eps=self.eps, swap=self.swap, reduction=self.reduction)
 
 
+class TripletMarginLossWithDistance(_Loss):
+    r"""Creates a criterion that measures the triplet loss given an input
+    tensors :math:`x1`, :math:`x2`, :math:`x3` and a margin with a value greater than :math:`0`.
```

Comment:
Probably want the names here to correspond with the names in argument list / documentation (e.g. "anchor", ...)

Reply:
Like "distance swap," I don't think "margin" is defined so it's not really clear what it should mean or why it must be greater than zero. Also you can just write zero or 0 without putting it in Tex.